graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -421,6 +423,30 @@ public DataAdapterApi updateAdapter(@Valid @ApiParam DataAdapterApi toUpdate) {
         return DataAdapterApi.fromDto(saved);
     }
 
+    @POST
+    @Path("adapters/validate")
+    @NoAuditEvent("Validation only")
+    @ApiOperation(value = "Validate the data adapter config")
+    public ValidationResult validateAdapter(@Valid @ApiParam DataAdapterApi toValidate) {
```

Comment:
Yes, anyway as I said, in the end the browser stops you from submitting that form, so it's just a bit of inconsistency :-)